src/main/java/de/tum/in/www1/artemis/service/FileUploadSubmissionService.java

```diff
@@ -88,11 +90,18 @@ public FileUploadSubmission handleFileUploadSubmission(FileUploadSubmission file
      *
      * @param exerciseId    - the id of the exercise we are interested into
      * @param submittedOnly - if true, it returns only submission with submitted flag set to true
+     * @param examMode - set flag to to ignore exam test run submissions
      * @return a list of file upload submissions for the given exercise id
      */
     @Transactional(readOnly = true)
-    public List<FileUploadSubmission> getFileUploadSubmissions(Long exerciseId, boolean submittedOnly) {
-        List<StudentParticipation> participations = studentParticipationRepository.findAllWithEagerSubmissionsAndEagerResultsAndEagerAssessorByExerciseId(exerciseId);
+    public List<FileUploadSubmission> getFileUploadSubmissions(Long exerciseId, boolean submittedOnly, boolean examMode) {
```

Comment:
`examMode`: personally I like Enums better for that, because it is more explicit which mode you are setting and it is easier for new developers to debug (does not require that much of domain specific knowledge, i.e. what is the contrary of `examMode`), just my opinion